cli/main.go

```diff
@@ -107,7 +107,12 @@ func main() {
 					UsageText: "draupnir config show",
 					Action: func(c *cli.Context) error {
 						fmt.Printf("Domain: %s\n", CONFIG.Domain)
-						fmt.Printf("Access Token: %s****\n", CONFIG.Token.AccessToken[0:10])
+						// Go doesn't appear to have a safe subslice operation...
+						if len(CONFIG.Token.AccessToken) < 10 {
```

Comment:
Definitely true that you can just look in the file - I'm just trying to avoid printing secrets to the terminal. Feels kinda bad, even if it's probably fine in this case.